includes/class-wc-google-analytics.php

```diff
@@ -475,35 +475,39 @@ public function add_to_cart() {
 
 		global $product;
 
-		// Add single quotes to allow jQuery to be substituted into _trackEvent parameters
-		$parameters = array();
-		$parameters['category'] = "'" . __( 'Products', 'woocommerce-google-analytics-integration' ) . "'";
-		$parameters['action']   = "'" . __( 'Add to Cart', 'woocommerce-google-analytics-integration' ) . "'";
-		$parameters['label']    = "'" . esc_js( $product->get_sku() ? __( 'ID:', 'woocommerce-google-analytics-integration' ) . ' ' . $product->get_sku() : "#" . $product->get_id() ) . "'";
+		if ( 'yes' === $this->ga_gtag_enabled ) {
+			$this->get_tracking_instance()->add_to_cart( $product );
+		} else {
```

Comment:
I personally prefer to reduce indented code blocks whenever possible. For example in this case the sole action is to run the class function instead and then exit the function, so if we add a return then we no longer need to move everything else in an else block.

```suggestion
		if ( 'yes' === $this->ga_gtag_enabled ) {
			$this->get_tracking_instance()->add_to_cart( $product );
			return;
		}
```

Reply:
Thanks @mikkamp - I've [adjusted the indentation](https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/240/commits/e44bb50b01307a9249a5e914405fd11d068b8c2a) as you suggested.